src/compiler/checker.ts

```diff
@@ -1473,8 +1476,6 @@ namespace ts {
             return undefined;
         }
 
-        // This is for caching the result of getSymbolDisplayBuilder. Do not access directly.
-        let _displayBuilder: SymbolDisplayBuilder;

```

Comment:
Ah I see; nice!. Perhaps it would be better to just make it a `var` declaration again.